diff --color -crB libao-1.2.0/src/plugins/pulse/ao_pulse.c libao-1.2.0.fix/src/plugins/pulse/ao_pulse.c
*** libao-1.2.0/src/plugins/pulse/ao_pulse.c	2014-01-27 18:02:05.000000000 +0100
--- libao-1.2.0.fix/src/plugins/pulse/ao_pulse.c	2024-12-24 11:56:28.490269514 +0100
***************
*** 31,36 ****
--- 31,37 ----
  #include <string.h>
  #include <signal.h>
  #include <limits.h>
+ #include <time.h>
  
  #include <pulse/pulseaudio.h>
  #include <pulse/simple.h>
